standard/clause_6_informative_text.adoc

```diff
@@ -533,7 +544,7 @@ For an in file variable reference to be declared, three conditions shall be met.
 *** 'search by absolute path' and 'search by relative path' shall be implemented;
 *** 'search by proximity' is deprecated by CF, and may be implemented or not.
 * Condition two: the attribute name is already interpreted as a URI, defining an entity, external to the file.
-* Condition three: the attribute name entity declares and `<rdfs:range>` of `<bald:Resource>`.
+* Condition three: the attribute name entity declares and `<rdfs:range>` of `<bald:Resource>` (or uses rdf:subClassOf linkage).
```

Comment:
Did you mean
```
Condition three: the attribute name entity declares an...
```

rdf:subClassOf linkage -> direct rdf:subClassOf linkage?

Reply:
typo fixed